File: app/src/main/java/com/master/kit/TestActivity/CalendarActivity.java
package com.master.kit.TestActivity;

import android.app.Activity;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import com.master.kit.R;
import com.master.kit.widget.CalendarView.CalendarView;

public class CalendarActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_calendar);
        CalendarView calendarView = (CalendarView) findViewById(R.id.cv_main);
        calendarView.setDate(2015,5);
    }
}

File: self-made/src/main/java/ooftf/com/widget/activity/CalendarActivity.kt
package ooftf.com.widget.activity
import android.os.Bundle
import com.alibaba.android.arouter.facade.annotation.Route
import com.ooftf.service.base.BaseSlidingActivity
import kotlinx.android.synthetic.main.activity_calendar.*
import ooftf.com.widget.R

@Route(path = "/widget/calendar")
class CalendarActivity : BaseSlidingActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_calendar)
        calendarView.setOnItemClickListener {
            toast(it.time.toLocaleString())
        }
    }
}
